### PR TITLE
Add alcohol wipes to the game

### PR DESCRIPTION
**Pull request by ISuckM8**
#### Summary

Content "Add alcohol wipes to the game"

#### Purpose of change

Always carry alcohol wipes with you, kids. It could save your life. Jokes aside, alcohol wipes are pretty common (As far as I know at least) disinfectants, I'm surprised we still don't have them in the game yet.

#### Describe the solution

Simply adds alcohol wipes, also known as antiseptic wipes, to post-apocalyptic New England. They spawn inside small cardboard boxes. They have a chance to spawn inside houses, doctor offices, and pharmacies. They're not as good as proper antiseptic but they're more common.

#### Describe alternatives you've considered

Not adding them.

#### Testing

My game works and it spawns properly.

#### Additional context

image

### CODE REVIEW

File: data/json/items/comestibles/med.json [hunk 1]
```diff
@@ -567,6 +567,23 @@
     "flags": [ "NO_INGEST", "WATER_DISSOLVE" ],
     "use_action": { "type": "heal", "disinfectant_power": 3, "bite": 0.95, "move_cost": 100 }
   },
+  {
+    "id": "alcohol_wipes",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": { "str_sp": "alcohol wipes" },
+    "description": "A type of wipes commonly used for treating contaminated wounds.",
```
Comment (chaohedgehogs) [inline data/json/items/comestibles/med.json:575]:
```suggestion
    "description": "A type of wet wipe saturated with alcohol and stored in a sterile package.  Commonly used for personal hygiene or disinfecting wounds.",
```

**Comment by I-am-Erk** [comment]
Alcohol wipes are close to useless as disinfectant... what does disinfectant_power 3 compare to?

**Comment by ISuckM8** [comment]
> Alcohol wipes are close to useless as disinfectant... what does disinfectant_power 3 compare to?

so, disinfectant_power 1? And they're supposed to be antiseptic wipes.

**Comment by I-am-Erk** [comment]
I mean, how does a value of 3 compare to other disinfecting items in the game?

**Comment by ISuckM8** [comment]
Disinfectant power 4 is proper antiseptic. 2 is soaked rags. Nothing is 3.

So, I'm gonna guess that I'll have to decrease the power to 1?

**Comment by I-am-Erk** [comment]
These probably aren't better than a disinfectant soaked cloth, I'd give them a 1 or 2 yeah.

**Pull request merged**